### PR TITLE
local fixes

### PR DESCRIPTION
**Pull request by tiwarishub**

### CODE REVIEW

File: src/configure/resources/messages.ts [hunk 1]
```diff
@@ -113,6 +113,7 @@ export class Messages {
     public static AdoDifferentTenantError = " One potential reason can be the AAD tenant of your Azure DevOps repository is different than the one you have been logged-in to VSCode. Open 'Command Pallete' and select 'Azure: Sign Out' and then Sign-In to other tenant by invoking 'Deploy to Azure' extension";
     public static ConfiguringPipelineFailed = "Configuring provisioning pipeline failed due to %s";
     public static CreatingSPN = "Creating SPN";
-    public static ConfiguringDraftPipeline = "Configuring draft pipeline";
+    public static GeneratingWorkflowFiles = "Generating workflow files";
     public static CreatingResourceGroup = "Creating resource group";
+    public static ConfiguringWorkflowAndDeployment = 'Configuring workflow and proceeding to deployment...';
```
Comment (anuragc617) [inline src/configure/resources/messages.ts:118]: It should be Configuring github workflow

**Review by anuragc617 — APPROVED** [review]

File: src/configure/resources/messages.ts [hunk 2]
```diff
@@ -113,6 +113,7 @@ export class Messages {
     public static AdoDifferentTenantError = " One potential reason can be the AAD tenant of your Azure DevOps repository is different than the one you have been logged-in to VSCode. Open 'Command Pallete' and select 'Azure: Sign Out' and then Sign-In to other tenant by invoking 'Deploy to Azure' extension";
     public static ConfiguringPipelineFailed = "Configuring provisioning pipeline failed due to %s";
```
Comment (anuragc617) [inline src/configure/resources/messages.ts:114]: Change this message as well to Configuring github workflow failed due to %s

File: src/configure/resources/messages.ts (hunk 1 above)
Comment (tiwarishub) [reply src/configure/resources/messages.ts]: done

File: src/configure/resources/messages.ts (hunk 2 above)
Comment (tiwarishub) [reply src/configure/resources/messages.ts:114]: Added

**Pull request merged**